### DISCUSSION TITLE
Second life battery related

### DISCUSSION THREAD

**Discussion by alihassan90** [pybamm-team/PyBaMM#3012] [General]
I am looking to simulate the battery degradation for retired battery. I am assuming the second life starts at 80% SOH. In your opinion which parameters do I need to change for simulating such an experiment. Also is there already tested parameter's file in PyBAMm repository that represents a use battery?

**Comment by brosaplanella** [comment]
We don't have parameter sets specifically for aged batteries, but you could probably take one of the parameter sets and run a few hundreds of cycles of a given degradation model and then use some of those states (e.g. porosity & SEI thickness, if considering SEI growth) for your new parameter set for an aged battery.

**Reply by alihassan90** [reply]
Thank you. When I print out these variables using pybamm.plot_summary_variables(sol) after running a long experiment the values are still the same as in the original parameters file. If they do get updated, how to access those updated values?

**Reply by brosaplanella** [reply]
Does your model include some degradation effects or is it the standard DFN/SPM?

**Reply by alihassan90** [reply]
i am using a standard DFN/SPM. do i need to make a degradation model? or there is one i can use in PyBAMm. My use case is an aged battery in the grid. I don't have much information on battery degradation but I ant to use PyBAMm for my simulations. what do you suggest? Thanks

**Reply by brosaplanella** [reply]
There is many options already built in PyBaMM, here you can find the link to the various options available: https://pybamm.readthedocs.io/en/latest/source/api/models/base_models/base_battery_model.html#pybamm.BatteryModelOptions

**Comment by Thiyaga2025** [comment]
hi @alihassan90,

i am trying to run simulations with an aged battery and looking for options. were you able to identify the parameters pertaining to aged cell. can you please help me with the running of such simulation. thanks and regards.
